src/sass/extends/_xx_Hero-Competing.scss

```diff
@@ -0,0 +1,187 @@
+/*--------------------------------------------------------------
+# Heroes
+
+1. Mixins
+2. Hero, Desktop
+3. Hero, Mobile
+4. Story Hero, Mobile
+5. Story Hero, Desktop
+--------------------------------------------------------------*/
```

Comment:
Useful remnant of the old markup for the hero which has been left within the UDS-Bootstrap package. This SASS partial is not compiled by the `_extends` stack from Bootstrap 4, therefore poses no "threat" to us with it's inclusion.